crates/guest-rust/src/lib.rs

```diff
@@ -10,6 +15,9 @@ pub use wit_bindgen_guest_rust_macro::{export, import};
 #[doc(hidden)]
 pub use bitflags;
 
+// Re-export things from liballoc for convenient use.
+pub use liballoc::{alloc, string, vec};
```

Comment:
Could these move into `mod rt` since that's the "internal stuff exported for the macro" module? (and would also avoid the need to rename `alloc` to `liballoc`)

Reply:
Done.